cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -78,16 +77,17 @@ func NewListerRegistry(allNode NodeLister, readyNode NodeLister, allPodLister Po
 }
 
 // NewListerRegistryWithDefaultListers returns a registry filled with listers of the default implementations
-func NewListerRegistryWithDefaultListers(kubeClient client.Interface, stopChannel <-chan struct{}) ListerRegistry {
-	allPodLister := NewAllPodLister(kubeClient, stopChannel)
-	readyNodeLister := NewReadyNodeLister(kubeClient, stopChannel)
-	allNodeLister := NewAllNodeLister(kubeClient, stopChannel)
-	podDisruptionBudgetLister := NewPodDisruptionBudgetLister(kubeClient, stopChannel)
-	daemonSetLister := NewDaemonSetLister(kubeClient, stopChannel)
-	replicationControllerLister := NewReplicationControllerLister(kubeClient, stopChannel)
-	jobLister := NewJobLister(kubeClient, stopChannel)
-	replicaSetLister := NewReplicaSetLister(kubeClient, stopChannel)
-	statefulSetLister := NewStatefulSetLister(kubeClient, stopChannel)
+func NewListerRegistryWithDefaultListers(informerFactory informers.SharedInformerFactory) ListerRegistry {
+	allPodLister := NewScheduledPodLister(informerFactory.Core().V1().Pods().Lister())
```

Comment:
Why assign *scheduled* pod lister to *all* pod lister variable? We're not using a separate sheduled pod lister anymore.

Reply:
oh, I made a incorrect renaming.